### ISSUE TITLE
TODO warning message includes excess leading whitespace

### ISSUE THREAD

**Issue by mattrubin** [realm/SwiftLint#792]
With `todo` warnings enabled, [this line][1]
``` swift
            // TODO: Scroll to the newly-expanded section
```
becomes this warning:
```
warning: Todo Violation: TODOs should be avoided             : Scroll to the n... (todo)
```

[1]: https://github.com/mattrubin/Authenticator/blob/2600e4cf027a87c979535742b59e1b18afc5140f/Authenticator/Source/TokenEntryForm.swift#L201

The TODO comment is indented to align with the code around it, but the [custom message for the todo rule][custom-message] is generated by simply stripping the "TODO" and "//" strings from the line. The leading whitespace and the colon I use in my TODO comments are left in the message, which is then truncated to leave very few useful characters.

[custom-message]: https://github.com/realm/SwiftLint/pull/765/files

The leading whitespace should be removed to allow more room for content in the message. Instead of trying to extract the important subset of the text from the line, the rule could simply strip the "//" and the preceding characters and give this warning:
```
warning: Todo Violation: TODO: Scroll to the newly-expanded section (todo)
```

**Comment by wbagdon** [comment]
Resolved with #801

**Comment by norio-nomura** [comment]
confirmed that this issue has been resolved.

**Comment by msaveleva** [comment]
swiftlint 0.12.0 – still have this issue.

**Comment by wbagdon** [comment]
It's fixed in master.
Will be included in the next release.
